### ISSUE TITLE
Fixed TextWireTest#testArrayTypes2

### ISSUE THREAD

**Issue by dpisklov** [OpenHFT/Chronicle-Wire#103]
Leftovers from #102

**Comment by RobAustin** [comment]
why ?

**Comment by dpisklov** [comment]
this only works for byte[] but not for e.g. char[], int[] etc.

**Comment by dpisklov** [comment]
```
    @Test
    public void testArrayTypes3() {
        Wire wire = createWire();
        wire.bytes().append("a: [ !type int[] ], b: !type String[], c: hi");

        assertEquals(String[].class, wire.read("b").typeLiteral());
        Collection<Class> classes = wire.read("a").typedMarshallable();
        assertArrayEquals(new Class[]{char[].class}, classes.toArray());
        assertEquals("hi", wire.read("c").text());
    }
```

**Comment by dpisklov** [comment]
See comment at the commit

**Comment by RobAustin** [comment]
please add a failing test

**Comment by dpisklov** [comment]
@RobAustin see above

**Comment by dpisklov** [comment]
Checked in the amended test

**Comment by RobAustin** [comment]
closing as all the tests pass

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.19.9](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.19.9), [BOM-2.19.111](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.111)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.19.9](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.19.9), [BOM-2.19.112](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.112)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.19.9](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.19.9), [BOM-2.19.116](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.116)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
